firebase/functions/getUserById.ts

```diff
@@ -0,0 +1,30 @@
+import { doc, getDoc } from 'firebase/firestore'
+import { BaseUser, User } from '../../src/types/types'
+import { db } from '../firebase'
+
+/*GetUsers by unique Id
+@author[Kevin]*/
+
+export const getUserById = async (
+  userId: string,
+): Promise<User | [] > => {
+  try {
+    // Reference the document by its ID
```

Comment:
Remove comment. Clyour code can explain what's happening for something this simple